etc/mpv.profile

```diff
@@ -16,11 +16,15 @@ include /etc/firejail/disable-passwdmgr.inc
 
 caps.drop all
 netfilter
+nogroups
 nonewprivs
 noroot
 protocol unix,inet,inet6
 seccomp
+tracelog
 
 # to test
+ipc-namespace
```

Comment:
I don't recall any problems with mpv specifically, but here I'd rather err on the side of stability since IIRC ipc-namespaces were significantly problematic for several people. Let's comment it.